Makefile.install

```diff
@@ -16,9 +16,9 @@
 # by dependencies between rules, so do *not* try overly clever things like
 # 'make world install' in one unique command
 
-ifeq ($(LOCAL),true)
+ifeq ($(COQPREFIX),local)
```

Comment:
I don't get how you can get in that state. When the prefix is not set, it doesn't get set to `local`, does it?

Reply:
The code now is `pr "COQPREFIX=%s\n" ((function None -> "local" | Some v -> v) !prefs.prefix)` 

The way the code is written makes extremely hard to reason about what is going on, this is what motivated me to start #14189 and friends, meanwhile this doesn't hurt too much IMHO but indeed should not be there for 8.14